DemoApp/DemoApp/DemoApp.csproj

```diff
@@ -82,6 +82,9 @@
 	    <SubType>Code</SubType>
```

Comment:
There is a UI issue in iOS. 
![Simulator Screen Shot - iPhone 14 - 2023-06-22 at 15 50 35](https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/5e0e8279-9fbb-490a-b668-9a5d2dd8e2bb)

Reply:
@ElishaSamPeterPrabhu Is this fixed?

Reply:
Yes